drupal8-example/web/sites/default/platformsh.common.settings.php

```diff
@@ -39,9 +39,8 @@
   if (!isset($settings['file_private_path'])) {
     $settings['file_private_path'] = $_ENV['PLATFORM_APP_DIR'] . '/private';
   }
-  if (!isset($config['system.file']['path']['temporary'])) {
-    $config['system.file']['path']['temporary'] = $_ENV['PLATFORM_APP_DIR'] . '/tmp';
-  }
+
+  $settings['file_temp_path'] = $_ENV['PLATFORM_APP_DIR'] . '/tmp';
```

Comment:
Hvorfor droppe `if`'en rundt om? Den er med ved `file_private_path` så hvis det er bevidst den er smidt væk bør det vel symetri-hensyn være ens for private og temp paths?